### PR TITLE
Update BosphorusSign

### PR DESCRIPTION
**Pull request by cleong110**
Updating BosphorusSign details including broken link, adding features for Kinectv2.

TODO a second pull request adding BosphorusSign22k, the updated release

https://github.com/sign-language-processing/sign-language-processing.github.io/issues/48 details.

### CODE REVIEW

File: src/datasets/BosphorusSign.json [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
+    "url": "https://ogulcanozdemir.github.io/bosphorussign22k/"
   },
-  "features": [],
+  "features": ["video:RGBD", "pose:Kinectv2", "gloss"],
   "notes": "Kinect v2",
   "language": "Turkish",
-  "#items": 636,
-  "#samples": "24,161 Samples",
+  "#items": 595,
+  "#samples": "22,670 Samples",
```
Comment (AmitMY) [inline src/datasets/BosphorusSign.json:12]: > BosphorusSign Turkish Sign Language corpus, which consists of 855 sign and p

https://aclanthology.org/L16-1220.pdf

Where is this info from? this is the number for Bosphorus22K, no?

**Review by AmitMY — CHANGES_REQUESTED** [review]

File: src/datasets/BosphorusSign.json (hunk 1 above)
Comment (cleong110) [reply src/datasets/BosphorusSign.json:12]: Table 3 of the 22k paper lists statistics for both datasets. 
333638579-10e469cc-8511-41f0-99a6-5416479ba427

Comment (cleong110) [reply src/datasets/BosphorusSign.json:12]: In the original BosphorusSign citation the number given is 855, not 636 or 595, we have:
* "The corpus contains 855 signs" in the conclusion section
* Table 2 talks about modalities/features
* Table 1 talks about _other_ datasets
* "We have collected 855 signs and phrase samples..." in the introduction section
* "When completed, the corpus will have at least six repetitions of each sign per-
formed by 10 signers, giving a wide variance to the data."

What I presume happened is that between the two papers they decided to trim down the "publicly available" data to 595 signs.

Comment (cleong110) [reply src/datasets/BosphorusSign.json:12]: It also seems that for whatever reason the "when completed... 10 signers" did not happen, as the newer citation lists only 6, and has this to say:

> Our dataset is based on the BosphorusSign (Cam-
goz et al., 2016c) corpus which was collected with the pur-
pose of helping both linguistic and computer science com-
munities. It contains isolated videos of Turkish Sign Lan-
guage glosses from three different domains: Health, finance
and commonly used everyday signs. Videos in this dataset
were performed by six native signers, as shown in Figure
1, which makes this dataset valuable for user independent
sign language studies.

"this dataset" I interpreted to mean that BosphorusSign, meaning that both BosphorusSign and BosphorusSign22k have the same number of signers, namely 6.

Comment (AmitMY) [reply src/datasets/BosphorusSign.json:12]: i think overall stats are more "correct" to use. thanks for checking!

**Review by AmitMY — APPROVED** [review]

**Pull request merged**